### PR TITLE
Update Source.cpp

### PR DESCRIPTION
**Pull request by Rocky04**
Decoupling the output from the measurements

### CODE REVIEW

File: Source.cpp [hunk 1]
```diff
@@ -54,18 +54,19 @@ void input_loop() {
...
-			if (input_times.size() > 1) {
-				input_times = { input_times.back() };
-				input_key = { input_key.back() };
-			}
+void measurements() {
+	if (TEST_POLLING_RATE == 2) {
+		for (int x = 1; x < input_times.size(); ++x) {
+			float result = float_millisecond(input_times[x] - input_times[x - 1]).count();
+			//if (result > 2 && result < 100) //for getting only the interesting timepoints
+				std::cout << result << ' ' << input_key[x] <<  'ms\n';
```
Comment (Rocky04) [inline Source.cpp]:
```suggestion
				std::cout << result << ' ' << input_key[x] <<  ' ms\n';
```

Comment (Rocky04) [inline Source.cpp:65]:
```suggestion
				std::cout << result << '  ms ' << input_key[x] <<  '\n';
```

**Comment by ad8e** [comment]
What you're describing is indeed the way I originally implemented it, but when I tested it, it wasn't necessary on any system measured, when the USB input is 1000 Hz. This is because the console input finished between the 1 ms timepoints, such that polling was able to pick up the next input immediately. Given this, I decided the instant feedback of seeing what was happening was much more important. The program is already inscrutable without copious reading, and making it even less user friendly would be a drastic problem.

I would be open to putting it as a non-default option with TEST_POLLING_RATE = 3 (or just repurposing TEST_POLLING_RATE = 1 since that is useless, and keeping the note of how the previous instant-output option shouldn't be done). This is because while I haven't yet seen a system for which console output is slow enough to interfere in the 1 ms gaps, one could exist.

Also, the "ms" text should be moved to either:
"ms" Time Key
Key Time "ms"
Having two letters next to each other is less readable. Or, output a note at the beginning that all times are in ms, instead of repeating "ms".

**Comment by Rocky04** [comment]
As mentioned multiple times now, an output to the console is very inconsistent. So you must eliminate it, if you want to claim even partial mentingful usable results. You even mentioned the problems because of this on the first method. So why the hell do you thing it didn't have a noticeable effect on method 2?

This output delay has nothing to do with the polling rate or the device itself, so I don't get it why are you even mentioning it.

To be honest, I don't care about your code. So take it or leave it. But I bet the results will be better, if you would apply my changes. I don't even want to spent time to build the project for my own.

_Why didn't you had add the project file from Visual Studio to Github? So others would quickly clone it completly without the trouble of creating all the things per hand. Just create of build folder for Win and add the project file into it, of course with the correct relative link to the used files._

**Comment by ad8e** [comment]
> As mentioned multiple times now, an output to the console is very inconsistent. So you must eliminate it, if you want to claim even partial mentingful usable results. You even mentioned the problems because of this on the first method. So why the hell do you thing it didn't have a noticeable effect on method 2?

Right, I just don't believe it is very inconsistent unless I see something indicating it is. You haven't provided any evidence for your claim and it seems you don't understand the existing code well, and why it works and why method 1 doesn't.

> To be honest, I don't care about your code. So take it or leave it. But I bet the results will be better, if you would apply my changes. I don't even want to spent time to build the project for my own.

Ok, I'll leave it.

> _Why didn't you had add the project file from Visual Studio to Github? So others would quickly clone it completly without the trouble of creating all the things per hand. Just create of build folder for Win and add the project file into it, of course with the correct relative link to the used files._

Because I can't ship GLFW. Even GLFW has trouble shipping its own binaries.

**Comment by Rocky04** [comment]
I already told you where the problem in your code is, but you don't react to it, or explain why you are thinking it's not a problem. The only things I read from you is stupid rhetoric.

_Write a simple program with two variables (delay and lastTime), save the current time to lastTime, print out some text with the value of the delay, calculate the current delay from the output from the current time and the lastTime, save calulated delay to delay and the current time to lastTime. Repeat this 1000 times and check the values. I bet you will find some high ones. Because your system will hire back your process from time to time for sure_

You aren't arguing with me, you didn't write any meaningful answerse there I could reply to it, so I don't see any meaning in still responding to you...

**Pull request closed without merging**